rdmo/projects/filters.py

```diff
@@ -13,6 +18,88 @@ class Meta:
         fields = ('title', 'catalog')
 
 
+class ProjectSearchFilterBackend(SearchFilter):
```

Comment:
This allows for http://localhost:8000/api/v1/projects/projects/?search=olga%20child to search in `title` and the `owners`.